From 227646dc0c2ae5134502e1bf0324b83d11e251c9 Mon Sep 17 00:00:00 2001
From: Antheas Kapenekakis <lkml@antheas.dev>
Date: Thu, 28 Aug 2025 11:21:41 +0200
Subject: [PATCH 095/112] drm/amdgpu: tweak min and max clocks of
 cyan_skillfish

Certain aftermarket cyan_skillfish boards that are
used for gaming have an expanded clock range that
is limited by this driver. Update it to reflect
the actual min and max clocks of the board.

Signed-off-by: Antheas Kapenekakis <lkml@antheas.dev>
---
 drivers/gpu/drm/amd/pm/swsmu/smu11/cyan_skillfish_ppt.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/pm/swsmu/smu11/cyan_skillfish_ppt.c b/drivers/gpu/drm/amd/pm/swsmu/smu11/cyan_skillfish_ppt.c
index 55401e6b2..4db6cc9a1 100644
--- a/drivers/gpu/drm/amd/pm/swsmu/smu11/cyan_skillfish_ppt.c
+++ b/drivers/gpu/drm/amd/pm/swsmu/smu11/cyan_skillfish_ppt.c
@@ -45,8 +45,8 @@
 #undef pr_debug
 
 /* unit: MHz */
-#define CYAN_SKILLFISH_SCLK_MIN			1000
-#define CYAN_SKILLFISH_SCLK_MAX			2000
+#define CYAN_SKILLFISH_SCLK_MIN			350
+#define CYAN_SKILLFISH_SCLK_MAX			2230
 
 /* unit: mV */
 #define CYAN_SKILLFISH_VDDC_MIN			700
-- 
2.52.0

